docs/endpoints.rst

```diff
@@ -829,20 +829,21 @@ Users Aggregation
 .. http:post :: /users/count
 
     Get ``aggregation`` statistics about OSM users. List of endpoints:
-    
+
     * **/count**
     * **/count/groupBy/(groupType)**
     * **/count/density**
     * **/count/density/groupBy/(boundary or tag or type)**
 
     * grouping type: one of boundary_, key_, tag_, type_.
-    
+
      .. note:: ``groupByKeys``, ``groupByKey`` and ``groupByValues`` are resource-specific parameters.
-    
+
     :query <other>: see above_
     :query groupByKeys: see key_
     :query groupByKey: see tag_
     :query groupByValues: see tag_
+    :query contributionType: filters users by type of contribution: ‘creation’, ‘deletion’, ‘tagChange’, ‘geometryChange’ or a combination of them; default: empty;
```

Comment:
I think we should be a bit more specific here, so that users can understand how this actually works. for example:

```suggestion
    :query contributionType: restricts result to counts of users which performed at least one of the given types of contribution; can be ‘creation’, ‘deletion’, ‘tagChange’, ‘geometryChange’ or a combination of them; default: empty;
```